doc/source/parquet.rst

```diff
@@ -21,29 +22,37 @@ File-Level Metadata
 The file-level metadata is stored as a JSON-encoded UTF-8 string under the "geo" key.
 For clarity, the following shows the unencoded JSON / `dict` structure:
 
-.. ipython:: python
+.. code-block:: none
 
     "geo": {
-    "columns": [...],  # JSON array of geometry columns
-    "creator": {
-        "library": "geopandas",
-          "version": "0.7.0",
-    },
-    "primary_column": "...",  # name of primary geometry column
-    "schema_version": "1.0.0"
+        "columns": [...],  # JSON array of geometry columns, see below
+        "creator": {
+            "library": "geopandas",
+            "version": "0.7.0",
+        },
+        "primary_column": "...",  # name of primary geometry column
+        "version": "1.0.0"
     }
 
 
 Column-Level Metadata
 ---------------------
 
+Each of the column entries in the ``"columns"`` field of the file-level metadata (see above)
+is another key-value object with following content:
+
+.. code-block:: none
+
+    {
+        "name": "<geometry column name>",
```

Comment:
@jorisvandenbossche how is this different than the name of the column that this metadata is attached to?

Reply:
Based on the current description, I understood that those column descriptions are put in an array in the "columns" field of the "geo" entry in the file-level metadata. 
So if that is the case, we need to mention the name of the column somewhere?

Now, parquet also has a concept of "column metadata", but I am not sure what would be the advantage of using that compared to keeping all information centralized in the file metadata (that's what the pandas metadata also do). I *think* you also have column metadata per row group, so this would then potentially be duplicated multiple times (https://parquet.apache.org/documentation/latest/#metadata).

Reply:
Ok - I see the confusion now.  At the file level, `columns` simply stores the names of the geometry columns, nothing more.

I used the column-level metadata to store column-specific information, like CRS - so it is the single source of truth for that level of information.

The same information could be stored at the file level using having a `columns` be a `list` of `dict`s with the column-level metadata.

One possible advantage (untested) of storing column-specific metadata at the column level is that you don't have to process it unless you read those columns (e.g., you have multiple geometry columns but only read one).